API_202404/src/main/java/com/example/api_202404/dto/SchoolDTOIncoming.java

```diff
@@ -0,0 +1,10 @@
+package com.example.api_202404.dto;
+
+import lombok.Data;
+
+@Data
+public class SchoolDTOIncoming {
```

Comment:
Incoming/Outgoing nelabai teko sutikti tokias sąvokas DTO pavadinimuose, dažniau naudojamos alternatyvos - Input/Output, Request/Response

Reply:
čia tiesiog random sugalvojau, nes man tas 'create' prefix ir be nieko labai maišėsi :D